### DISCUSSION TITLE
Is there anyway to import the "starttime" of the alert as a lebel into prometheus

### DISCUSSION THREAD

**Discussion by ghost** [prometheus-operator/prometheus-operator#4459] [Q&A]
I am trying to find a way to add starttime of the alert as a label of prometheus or try to get current timestamp as a label, anyone can advise how to do it?

**Comment by paulfantom** [comment]
Timestamps shouldn't be labels as they can lead to cardinality explosions. Consider adding timestamps as separate metrics.

You can look at `ALERTS{}` series to find which alert fired when and for how long.
